### ISSUE TITLE
Is it really BGR?

### ISSUE THREAD

**Issue by see--** [qubvel/classification_models#4]
Hello qubvel,
thanks for the great repo. Are you sure about the preprocessing for resnets (i.e. using BGR)? I get better results when I use RGB:
```
import numpy as np
from skimage.io import imread
from keras.applications.imagenet_utils import decode_predictions

from classification_models.classification_models import ResNet18
from classification_models.classification_models.resnet import preprocess_input

# read and prepare image
img_rgb = imread('imgs/tests/Lion.jpg')

for order in ['rgb', 'bgr']:
  print("Order: ", order.upper())
  x = preprocess_input(img_rgb, size=(224, 224), BGRTranspose=order == 'bgr')
  x = np.expand_dims(x, 0)

  # load model
  model = ResNet18(input_shape=(224, 224, 3), weights='imagenet', classes=1000)

  # processing image
  y = model.predict(x)

  # result
  print(decode_predictions(y))
```
Output:
```
Order:  RGB
[[('n02129165', 'lion', 0.9629949), ('n02115913', 'dhole', 0.01578529), ('n02112137', 'chow', 0.0087916795), ('n02114855', 'coyote', 0.0038442488), ('n02114712', 'red_wolf', 0.0014433597)]]
Order:  BGR
[[('n02129165', 'lion', 0.3915644), ('n02112137', 'chow', 0.10712016), ('n02114855', 'coyote', 0.09258213), ('n02114367', 'timber_wolf', 0.08412136), ('n02091467', 'Norwegian_elkhound', 0.027228368)]]
```

![lion](https://user-images.githubusercontent.com/11165293/47703504-c8c1ac00-dc20-11e8-8f1d-3932ecf5ca70.jpg)

**Comment by qubvel** [comment]
Hi @see-- !

I have converted models from MXNet framework, so rely on [this](https://github.com/Microsoft/MMdnn/blob/c94bf33d71cb1cbebe289af57814d44c37cafb8e/mmdnn/conversion/examples/imagenet_test.py#L134) and [this](https://github.com/Microsoft/MMdnn/blob/c94bf33d71cb1cbebe289af57814d44c37cafb8e/mmdnn/conversion/examples/imagenet_test.py#L294) code, where BGR transpose set as `True` for image for all resnet models. Also, I have same classification results as MXNet models, so it is not a conversion error.

However, thank you for report, may be it is good idea to test it with imagenet validation set to identify right preprocessing function!

**Comment by see--** [comment]
Indeed, your code is correct. I still wonder if they mixed the conversion in the MMdnn repo. Maybe I get some time to check.
